frontend/src/reducer/workspaceList.js

```diff
@@ -28,24 +30,25 @@ export const serializeWorkspaceListProps = {
   members: 'memberList'
 }
 
-export const serializeWorkspace = workspace => {
+export const serializeUserConfig = m => {
```

Comment:
rename to serializeUserWorkspaceConfig

Reply:
serializeUserWorkspaceConfig is another function used to seralize userWorkspaceConfig object with user, workspace and config value, but in this case it only serialize a user and it's config

Reply:
agreed